theme/static/js/hs_file_browser.js

```diff
@@ -422,11 +425,17 @@ function bindFileBrowserItemEvents() {
 
 function showFileTypeMetadata(){
      var logical_file_id = $("#fb-files-container li.ui-selected").attr("data-logical-file-id");
-     if (logical_file_id && logical_file_id.length == 0){
+     if (!logical_file_id || (logical_file_id && logical_file_id.length == 0)){
          return;
      }
      var logical_type = $("#fb-files-container li").children('span.fb-logical-file-type').attr("data-logical-file-type");
+     if (!logical_type){
```

Comment:
These avoid erroneous incomplete calls to the backend for non-existent information.